newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/index.js

```diff
@@ -217,22 +209,6 @@ const GetStartedSection = ({
     [onLoginWithProvider]
   );
 
-  React.useEffect(
-    () => {
-      if (step === 'welcome' && profile && profile.survey) {
-        setStep('recommendations');
-      } else if ((step === 'login' || step === 'register') && profile) {
-        setStep(profile.survey ? 'recommendations' : 'survey');
-      } else if (!(step === 'login' || step === 'register') && !profile) {
-        setStep('welcome');
-      }
-      // Only show user chip when the user is logged in and can see the recommendations.
-      // In any other case, we don't want to distract them from completing the survey.
-      showUserChip(step === 'recommendations' && !!profile && !!profile.survey);
```

Comment:
should we still hide the userShip if filling the survey? (if not, is this prop still used?)

Reply:
I think this was useful when the survey was the first thing you saw, too avoid complexifying the interface for the user with "too many buttons". But now, it's just something that you choose to do, after logging. No reason to hide your profile badge, it may even be considered as a bug (why this thing is disappearing while I do my survey? I still want to know who I am and if this is too many buttons, that's a problem of the interface, not the survey)

Reply:
(I've cleaned the props for the user ship so that there is none to hide them anymore)